packages/react-component-library/src/components/NumberInput/ClearButton.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import { IconCancel } from '@royalnavy/icon-library'
+import { selectors } from '@royalnavy/design-tokens'
+import styled, { css } from 'styled-components'
+
+const { color, spacing } = selectors
+
+interface ClearButtonProps {
+  isCondensed: boolean
+  isDisabled: boolean
+  onClick: (event: React.MouseEvent<HTMLButtonElement>) => void
+}
+
+interface StyledButtonProps {
+  isCondensed: boolean
+}
+
+const StyledButton = styled.button<StyledButtonProps>`
+  background-color: ${color('neutral', 'white')};
+  border: none;
+  color: ${color('neutral', '300')};
+  cursor: pointer;
+  position: absolute;
+  top: ${spacing('2')};
```

Comment:
These styles tighten up the focus around the clear button:

```css
top: 50%;
transform: translateY(-50%);
height: 1rem;
```

<img width="168" alt="Screenshot 2020-10-28 at 09 47 28" src="https://user-images.githubusercontent.com/48086589/97420113-cc1cca80-1902-11eb-9c1a-c7010fe5ba83.png">